webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -861,6 +867,20 @@ public EntityMutationResponse createOrUpdate(AtlasEntitiesWithExtInfo entities,
         }
     }
 
+    private void validateAttributeLength(final List<AtlasEntity> entities) throws AtlasBaseException {
+        //Predicate to check attribute value exceeding length
+        Predicate<Map.Entry<String, Object>> predicateOfAttributeValueMoreThan100K = attribute ->
```

Comment:
Change name of predicate to predicateOfAttributeLengthExceedingLimit

Reply:
Thanks. Renamed predicate